src/O1Heap.h

```diff
@@ -5,42 +5,44 @@
  * Contributors: https://github.com/107-systems/107-Arduino-Cyphal/graphs/contributors.
  */
 
-#ifndef ARDUINO_O1_HEAP_HPP_
-#define ARDUINO_O1_HEAP_HPP_
+#ifndef ARDUINO_O1_HEAP_H_
+#define ARDUINO_O1_HEAP_H_
 
 /**************************************************************************************
  * INCLUDE
  **************************************************************************************/
 
 #include <o1heap/o1heap.h>
 
+/**************************************************************************************
+ * DEFINE
+ **************************************************************************************/
+
+#define CYPHAL_DECLARE_HEAP(name,size) \
+  static uint8_t name[size] __attribute__ ((aligned (O1HEAP_ALIGNMENT)))
+
 /**************************************************************************************
  * CLASS DECLARATION
  **************************************************************************************/
 
-template <size_t HEAP_SIZE>
 class O1Heap
 {
 public:
-
-  O1Heap();
+  O1Heap(uint8_t * heap_ptr, size_t const heap_size)
```

Comment:
I think this should be non-copyable and non-movable.

Reply:
I'm not sure I fully get what you mean by this. This wrapper class is a private member of `Node`, hence it does not really matter its copyable or moveable as the user does not get to use it.

Reply:
Eliminated the wrapper class altogether in 77aace9bf0de506ce07f7fb5504346de1c45500f.